manifests/3/3u/3utools/3.03/3u.3utools.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: 3u.3utools
+PackageVersion: 3.03
+Installers:
+- InstallerUrl: https://url.3u.com/zmAJjyaa
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: 8FAF1527DCE8FAAD70226659157F81374DBC69D4CB13BEE4B06B6082AAE23440

```

```suggestion
  InstallerSha256: A665E457EEBF33F29D481F8E4FF31570CA7074C272FDEDEF9AAE7F7CA0956EC6
```

Apologies for posting twice and deleting once. Having a small tools issue.